enclave_manager/avalon_enclave_manager/singleton/enclave_manager.py

```diff
@@ -435,11 +435,11 @@ def create_json_worker(enclave_data, config):
         config.get("WorkerConfig")["WorkOrderNotifyUri"]
     details_info["receiptInvocationUri"] = \
         config.get("WorkerConfig")["ReceiptInvocationUri"]
-    details_info["workOrderInvocationAddress"] = config.get(
-        "WorkerConfig")["WorkOrderInvocationAddress"]
-    details_info["receiptInvocationAddress"] = config.get(
-        "WorkerConfig")["ReceiptInvocationAddress"]
-    details_info["fromAddress"] = config.get("WorkerConfig")["FromAddress"]
+    # workOrderInvocationAddress/receiptInvocationAddress/fromAddress
+    # are not being used as of now. So removed from config.
```

Comment:
Does that mean that these parameters will be used in future or these are yet to be cleaned from source code?

Reply:
Reverted this. https://github.com/hyperledger/avalon/pull/424#discussion_r412715495. More logical to have empty configs rather than hard coding if not doing a complete cleanup otherwise.